res/macos/run-space-acres.sh

```diff
@@ -1,17 +1,8 @@
 #!/bin/bash
 
 cd "$(dirname "$0")"
-LAUNCH_DIR="$(pwd)"
+EXECUTABLE_PATH="$(pwd)"
 
-loaders=/tmp/saloaders.cache
-if [ $loaders ]; then
-  rm $loaders
-fi
+export GDK_PIXBUF_MODULE_FILE="$EXECUTABLE_PATH/../Resources/lib/gdk-pixbuf-2.0/2.10.0/loaders.cache"
 
-cp ../Frameworks/loaders.cache $loaders
-sed -i '' "s|PREFIX|$(pwd)/../Frameworks|g" $loaders
-
-export GDK_PIXBUF_MODULEDIR="$LAUNCH_DIR/../Frameworks/loaders"
-export GDK_PIXBUF_MODULE_FILE=$loaders
-
-"$LAUNCH_DIR/space-acres"
\ No newline at end of file
+exec "$EXECUTABLE_PATH/space-acres"
```

Comment:
This makes much more sense, but now I'm wondering if we can get rid of `GDK_PIXBUF_MODULE_FILE`. Can't we know statically where it will be stored or have some kind of installer hook to fix it after extraction?

Reply:
@nazar-pc We have a solution is to patch the binary (`libgdk_pixbuf-2.0.0.dylib`), after that we don't need to use `run-space-acres.sh` anymore. What do you think?

Reply:
Depends on what kind of patching are we talking about. If something like what we already do already then fine, otherwise I'm okay leaving it as is. The key thing is that we no longer copy to `/tmp` and patch stuff, which is the important thing.